docs/resources/Audit_Log.md

```diff
@@ -2,22 +2,24 @@
 
 ## Audit Logs
 
-Whenever an admin action is performed on the API, an entry is added to the respective guild's audit log. You can specify the reason by attaching the `X-Audit-Log-Reason` request header. This header supports url encoded utf8 characters.
+When an administrative action is performed in a guild, an entry is added to its audit log. Viewing audit logs requires the `VIEW_AUDIT_LOG` permission and can be fetched by apps using the [`GET /guilds/{guild.id}/audit-logs` endpoint](#DOCS_RESOURCES_AUDIT_LOG/get-guild-audit-log), or seen by users in the guild's **Server Settings**. All audit log entries are stored for 45 days.
```

Comment:
aren't they stored for 90 days? i've always heard it was 90 and not 45

Reply:
hmmm when I was looking in the code it looked like it was 45, but let me check again

Reply:
I believe it used to be 90 but was changed to 45 years ago

Reply:
It looks like only 45 at the moment—checked in some internal docs and verified in the client for a few different guilds.